backend/events.json

```diff
@@ -0,0 +1,56 @@
+[
+    {
+        "id": "001",
+        "lesson": "html-1",
+        "date": "01-09-2018",
```

Comment:
This is probably more relevant when creating the database, but it is probably a good idea to model the event time as well as the date. In fact it might be good to have an event start date/time and end date/time